src/dcap_provider.h

```diff
@@ -6,6 +6,9 @@
 #define PLATFORM_QUOTE_PROVIDER_H
 
 #include <stdint.h>
+#include <sstream>
+
+using namespace std;

```

Comment:
Is this necessary? I thought because you explicitly use `std::stringstream` with `std::` prefix, you don't need to use the namespace?

Reply:
Removed :)